### PR TITLE
feat resetpassword

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added resetpassword feature

### CODE REVIEW

File: Forget/Forget.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        box-shadow: var(--box-shadow);
+}
+input{
+width:90%;
+padding: 0.6rem 0;
+border-radius:3px;
+margin:5px 0 1rem 8px ;
+border:1px solid var(--primary-color);
+color:var(--text-color)
+}
```
Comment (kshitijthatte) [inline Forget/Forget.css:22]: It's recommended to avoid styling the HTML elements directly.

File: Forget/Forget.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <li ><a href="/Logout/Logout.html"><i class="fas fa-user"></i></a></li>
+            </ul>
+        </div>
+       
+  </header>
+        <div class="forget__box">
+            <h3>PASSWORD RESET</h3>
+           <input type= text placeholder="E-mail">
+            <h4>**You will recieve an reset link in email id mentioned above</h4>
+            <div class="auth__box-sub">
```
Comment (kshitijthatte) [inline Forget/Forget.html:35]: You can give padding to this h4 text for a better consistent look.

File: Forget/Forget.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.forget__box{
+    position: relative;
```
Comment (kshitijthatte) [inline Forget/Forget.css:2]: You can use some code formatter like prettier, for more readable code.

**Review by kshitijthatte — APPROVED** [review]
Looks good!

**Review by someshtiwarii — APPROVED** [review]
Other than what Kshitij has suggested, rest looks good !

**Review by srishti-maurya — APPROVED** [review]

File: Forget/Forget.css (hunk 3 above)
Comment (Shrinkhlarajpoot) [reply Forget/Forget.css:2]: okay i will look at it..Thanks

**Pull request merged**